javascript/javascript2/week1/hmw2.js

```diff
@@ -0,0 +1,25 @@
+const danishString = ["Jeg har en blå bil"];
+const danishString2 = ["Blå grød med røde bær"];
+function danish(danishString) {
+  const string = danishString.join();
```

Comment:
Try to add meaningful variable names.
```suggestion
  const danishWords= danishString.join();
```